compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -1782,12 +1782,17 @@ static LogicalResult setRootDefaultConfig(IREE::GPU::TargetAttr target,
                                                preferredSubgroupSize);
 }
 
+/// Returns true if it's MatVec like i.e., either the bound of M dim = 1, or the
+/// M dim isn't present.
 static bool isMatvecLike(linalg::LinalgOp linalgOp) {
-  if (linalgOp.getNumParallelLoops() != 2)
+
+  if (!llvm::is_contained({1, 2}, linalgOp.getNumParallelLoops())) {
     return false;
```

Comment:
I think the requirement is really that there's exactly one parallel dimensions of size != 1, no?

Reply:
Yes, that means it can also support cases where N = 1.

Reply:
Can we implement it this way to make it general?

Reply:
Sure.

Reply:
Done, PTAL.